### ISSUE TITLE
Сan't erase 1 pixel

### ISSUE THREAD

**Issue by fortey** [AGulev/drawpixels#13]
`drawpixels.pixel(self.buffer_info, self.touch_pos.x, self.touch_pos.y, 0, 0, 0, 0)` doesn't work

**Comment by AGulev** [comment]
could you pls provide a test project?

**Comment by Morgerion** [comment]
Indeed, it does not work, since if alpha==0 then the pixel is not recorded. This follows from the code of the mixpixel() function.
It would be nice to have a function like pixel_raw() that just writes RGBA without any alpha channel checks.
I use this to fill technical textures where the alpha channel is just one of the data layers.
